security.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Security"
+---
+
+{% toc %}
+
+
+## Security Updates
+
+This section lists fixed vulnerabilities in Flink.
+
+<table class="table">
+	<thead>
+		<tr>
+			<th style="width: 20%">CVE ID</th>
+			<th style="width: 30%">Affected Flink versions</th>
+			<th style="width: 50%">Notes</th>
+		</tr>
+	</thead>
+	<tr>
+		<td>
+			<a href="https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-1960">CVE-2020-1960</a>
+		</td>
+		<td>
+			1.1.0 to 1.1.5, 1.2.0 to 1.2.1, 1.3.0 to 1.3.3, 1.4.0 to 1.4.2, 1.5.0 to 1.5.6, 1.6.0 to 1.6.4, 1.7.0 to 1.7.2, 1.8.0 to 1.8.3, 1.9.0 to 1.9.2, 1.10.0
+		</td>
+		<td>
+			<a href="https://github.com/apache/flink/commit/804ae70024bf8be7c0c7093d02addb080c318662">Fixed in commit 804ae70024bf8be7c0c7093d02addb080c318662</a> <br>
+			Users are advised to upgrade to Flink 1.9.3 or 1.10.1 or later versions.
```

Comment:
This not only excludes some information from the actual advisory we published (at the very least changing the configuration should be listed as an option), but for users upgrading to 1.9.3 the commit hash is also wrong, and thus of questionable value (also applies to other entries). Maybe just omit the commit hash entirely.

Reply:
I removed the commit sha.